templates/apps/wry/Cargo.toml.hbs

```diff
@@ -11,6 +11,21 @@ crate-type = ["staticlib", "cdylib", "rlib"]
 name = "{{app.name}}-desktop"
 path = "gen/bin/desktop.rs"
 
+[package.metadata.cargo-android]
+app-activity-name = ".MainActivity"
+app-dependencies = [
+"androidx.webkit:webkit:1.4.0",
+"androidx.appcompat:appcompat:1.5.0",
+"com.google.android.material:material:1.6.1",
+]
+project-dependencies = ["org.jetbrains.kotlin:kotlin-gradle-plugin:1.6.10"]
+app-plugins = ["org.jetbrains.kotlin.android"]
+app-permissions = ["android.permission.INTERNET"]
+app-theme-parent = "Theme.MaterialComponents.DayNight.DarkActionBar"
+
+[package.metadata.cargo-apple.ios]
+frameworks = ["WebKit"]
```

Comment:
If we add this, we should remove `Wbkit.framework` in project.yaml

Reply:
yeah, that's what I am going to do in the next PR

Reply:
actually let me remove it from this PR and I will add it back in the other PR